### ISSUE TITLE
Window filter focus order across spaces

### ISSUE THREAD

**Issue by saagarjha** [Hammerspoon/hammerspoon#1260]
I'm trying to use a cross-space window filter:

```lua
windowFilter = hs.window.filter.new():setCurrentSpace(nil)
```

When I call `windowFilter:getWindows()`, I'd expect it to return windows in order of their last focused time; for example, if I have the following window set (in order of most recently focused):

1. Space 1:
    - App 1 Window 1
    - App 1 Window 2
    - App 2 Window 1
2. Space 2:
    - App 3 Window 1

and I start with App 1 Window 1, and swipe to Space 2, then swipe back, I'd assume the window filter would give the following:

1. App 1 Window 1
2. App 3 Window 1
3. App 1 Window 2
4. App 2 Window 1

However, I instead get the windows grouped by app/space:

1. App 1 Window 1
2. App 1 Window 2
3. App 2 Window 1
3. App 3 Window 1

The works as expected if the windows are all in the same space. Is there any way to force the window filter to completely ignore spaces (or is this a bug?)

**Comment by lowne** [comment]
@saagarjha see #1460 
TLDR: `windowFilter = hs.window.filter.new():setCurrentSpace(nil):keepActive()`
